### DISCUSSION TITLE
Repo of Website

### DISCUSSION THREAD

**Discussion by BernhardSchlegel** [quasarframework/quasar#9188] [Ideas / Proposals] [answered]
Hey there,

is the website (https://quasar.dev/) available as repo? 

Best,
Bernhard.

**Comment by metalsadman** [comment, marked as answer]
> Hey there,
> 
> is the website (https://quasar.dev/) available as repo?
> 
> Best,
> Bernhard.

https://github.com/quasarframework/quasar/tree/dev/docs

**Reply by BernhardSchlegel** [reply]
thank you !
